site/app/templates/polls/NewPollPage.twig

```diff
@@ -27,7 +27,7 @@
             <label for="image-file">
                 Optional Image:
             </label>
-            <input type="file" id="image-file" name="image_file"/>
+            <input type="file" id="image-file" name="image_file" accept="image/jpeg, image/jpg, image/gif, image/png"/>
```

Comment:
You can put `accept="image/*"` instead of manually putting each image type, I think its cleaner since we're accepting every image type anyways

Reply:
From what I saw, `isValidImage()` checks if the picture is of MIME type image, and if the MIME sub-type is one of those in `FileUtils::ALLOWED_IMAGE_TYPE`. I took a look at that constant at this line: https://github.com/Submitty/Submitty/blob/dbee91d58f4918b5c9fbc60bae2e5989ac959f73/site/app/libraries/FileUtils.php#L15
So I figured that specifying for jpeg, jpg, gif, and png types instead of "image/*" would prevent potential conflicts between what files the pop-up file browser allows you to select and what the function `isValidImage()` accepts. Did I misunderstand, or maybe do you think that this should be changed anyways?

Reply:
Yeah that's true, I'm looking at the list of image types now and its more than the 4 we support so keeping it the same would work with the backend validation whatever you choose